doc/spec/estimation/dml.rst

```diff
@@ -95,95 +95,8 @@ one can impose several extensions to the matrix of parameters :math:`\alpha`, su
 constraints on the matrix :math:`\alpha`. Even under regularized second stage estimation, there is benefit in using the Double ML approach 
 as it still renders the MSE of second stage estimation robust to first stage errors (see e.g.  [Chernozhukov2017]_, [Chernozhukov2018]_). 
 
-API and Pseudocode for General Double ML Estimator
```

Comment:
Removing this is good.  Note that we can link directly to the actual documentation for the classes now if that's desired, with syntax like ``:py:class:`~econml.dml.DMLCateEstimator` ``

Reply:
Agreed we should

Reply:
Done. I am referring to the actual claases now.